addons/web/static/src/core/utils/draggable_hook_builder.js

```diff
@@ -580,6 +586,15 @@ export function makeDraggableHook(hookParams) {
                 const xRect = ctx.current.scrollParentXRect;
                 const yRect = ctx.current.scrollParentYRect;
 
+                // getBoundingClientRect() (used in getRect()) gives the
+                // distance from the element's top to the viewport, excluding
+                // scroll position. Only the <html> element accounts for
+                // scrollTop.
+                const scrollParentYEl = ctx.current.scrollParentY;
+                if (scrollParentYEl?.nodeName === "HTML") {
```

Comment:
- Why not comparing to `ownerDocument.scrollingElement` here?
- How is it possible to have `scrollParentYEl` falsy?

Reply:
> * Why not comparing to `ownerDocument.scrollingElement` here?

Was not sure if `ownerDocument.scrollingElement` is always an HTML element

> * How is it possible to have `scrollParentYEl` falsy?

This is the case when `closestScrollableY` used in `getScrollParents` returns `NULL` (no vertical scrollbar on the page)